### PR TITLE
Initial commit for Akumina types (see akumina.github.io)

### PR DESCRIPTION
**Pull request by MarshHawk**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [X] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [X] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [X] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [X] `tslint.json` should be present and it shouldnt have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [X] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,6 +2,5926 @@
 # Add yourself to the "Definitions by:" list instead.
 # See https://github.com/DefinitelyTyped/DefinitelyTyped#edit-an-existing-package
 
+<<<<<<< HEAD
```
Comment (amcasey) [inline .github/CODEOWNERS]: This file is updated automatically by a tool - please don't edit it directly.

**Review by amcasey — CHANGES_REQUESTED** [review]
The underlying package appears to already contain a .d.ts file.  Is the intention to host it here instead?

File: .github/CODEOWNERS (hunk 1 above)
Comment (MarshHawk) [reply .github/CODEOWNERS]: I think this is fixed now, thanks

Comment (MarshHawk) [reply .github/CODEOWNERS]: Also, yes, the intention is to deprecate the akumina-core NPM package and just release the types package for now...

**Comment by MarshHawk** [comment]
Addressed issues, thanks

**Comment by MarshHawk** [comment]
@amcasey Hey, I think we addressed the issues. Just wondering if the wait is due to additional fixes/ our approach or if you're just flat out?...

**Comment by amcasey** [comment]
@MarshHawk Oh no!  I completely missed the ping.  Sorry about that.  Looking now.

File: types/akumina-core/tslint.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "npm-naming": false,
```
Comment (amcasey) [inline types/akumina-core/tslint.json]: This appears to be unused.

File: types/akumina-core/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,1471 @@
...
+        class Calendar {
+            constructor(options: any);
+        }
+        class CompanyCalendar {
+            constructor(options: any);
+        }
+    }
+
+    namespace Digispace {
+        class ConfigurationContext {
```
Comment (amcasey) [inline types/akumina-core/index.d.ts]: I assume you know from the lint messages that a class with only static members doesn't make much sense but prefer to keep it this way.

**Review by amcasey — APPROVED** [review]

File: types/akumina-core/index.d.ts (hunk 3 above)
Comment (MarshHawk) [reply types/akumina-core/index.d.ts]: yes, yes, just deferring that improvement for the moment, thanks.

**Pull request merged**